Source/Loaders/ObjLoader.cpp

```diff
@@ -0,0 +1,104 @@
+#include <map>
+#include <string>
+#include <iostream>
+#include <Loaders/ObjLoader.h>
+
+using namespace std;
+
+static string VERTEX_LABEL = "v";
+static string TEXTURE_COORDINATE_LABEL = "vt";
+static string FACE_LABEL = "f";
+
+/**
+ * ObjLoader
+ * ---------
+ */
+ObjLoader::ObjLoader(const char* path) {
```

Comment:
Yeah this is exactly how I would have gone about parsing the .obj, except you did it in a much clearer and concise way.